aten/src/THCUNN/generic/SpatialConvolutionMM.cu

```diff
@@ -232,6 +234,8 @@ void THNN_(SpatialConvolutionMM_updateOutput)(
     THCudaBlas_Hgemm(
     #elif defined(THC_REAL_IS_DOUBLE)
     THCudaBlas_Dgemm(
+    #elif defined(THC_REAL_IS_BFLOAT16)
+    THCudaBlas_Bgemm(
```

Comment:
This will not be defined for CUDA since `THC_REAL_IS_BFLOAT16` will defined only for ROCm as the bfloat16 header is only included for ROCm. https://github.com/ROCmSoftwarePlatform/pytorch/blob/a1ccdf83f6e0854166bd9118c3513d4b79a23c72/aten/src/THCUNN/SpatialConvolutionMM.cu#L12-L15